packages/vats/test/localchain.test.js

```diff
@@ -0,0 +1,156 @@
+import { test as anyTest } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
+import { reincarnate } from '@agoric/swingset-liveslots/tools/setup-vat-data.js';
+import { E } from '@endo/far';
+import { M, matches } from '@endo/patterns';
+import { makeDurableZone } from '@agoric/zone/durable.js';
+import { AmountMath, AssetKind, makeIssuerKit } from '@agoric/ertp';
+import { buildRootObject } from '../src/vat-bank.js';
+import { prepareLocalChainTools } from '../src/localchain.js';
+
+/** @type {import('ava').TestFn<ReturnType<makeTestContext>>} */
+const test = anyTest;
+
+const makeTestContext = async t => {
+  const { fakeVomKit } = reincarnate({ relaxDurabilityRules: false });
+  const provideBaggage = key => {
+    const root = fakeVomKit.cm.provideBaggage();
+    const zone = makeDurableZone(root);
+    return zone.mapStore(`${key} baggage`);
+  };
+  const baggage = provideBaggage('localchainTest');
+  const bankVat = E(buildRootObject)(null, null, baggage);
+
+  const zone = makeDurableZone(baggage);
+
+  /** @type {undefined | ERef<import('../src/types.js').BridgeHandler>} */
+  let bankHandler;
+
+  /** @type {import('../src/types.js').ScopedBridgeManager} */
+  const fakeBridgeManager = zone.exo('fakeBankBridgeManager', undefined, {
```

Comment:
this will be useful outside this test. I've got one in https://github.com/Agoric/agoric-sdk/blob/f4cbbf754ea22e57f34a25c0919acd07d7ad2e4a/packages/orchestration/test/supports.ts#L58

fine to duplicate for now but at some point let's see how we can factor these for re-use